src/components/Login.js

```diff
@@ -41,6 +42,12 @@ export const validateJWT = ({ jwt }) => {
   return isCurrent && isApproved && validatedPayload;
 };
 
+const initProfile = async (api, user, egoId) => {
+  const x = await createProfile(api)({ ...user, egoId });
+  createExampleQueries(api, egoId);
```

Comment:
potential race condition here? maybe `await createExampleQueries()` (see note below bout async implementation of `createExampleQueries` :)